File: app/src/main/java/com/proxy/shadowsocksr/Consts.java
package com.proxy.shadowsocksr;

import android.annotation.SuppressLint;

public class Consts
{
    //Default ss profile
    public final static String defaultIP = "please.change.me";
    public final static String defaultCryptMethod = "chacha20";
    public final static int defaultRemotePort = 233;
    public final static int defaultLocalPort = 1093;
    //SSR
    public final static String defaultTcpProtocol = "origin";
    public final static String defaultObfsMethod = "plain";
    //
    //TODO: should not use hard-coded
    @SuppressLint("SdCardPath") public final static String baseDir
            = "/data/data/com.proxy.shadowsocksr/";
    //
    //
    public final static int STATUS_CONNECTED = 0;
    public final static int STATUS_FAILED = 1;
    public final static int STATUS_DISCONNECTED = 2;
    //
    public final static String TEMP_PREF_NAME = "dvy78dfv76dftv67dtv6tg6fg6vg";
    //
    public final static String lineSept = System.getProperty("line.separator");
}

File: app/src/main/java/com/proxy/shadowsocksr/Consts.kt
package com.proxy.shadowsocksr

import android.annotation.SuppressLint

object Consts
{
    //Default ss profile
    @JvmStatic const val defaultLabel = "Sample"
    @JvmStatic const val defaultIP = "please.change.me"
    @JvmStatic const val defaultCryptMethod = "chacha20"
    @JvmStatic const val defaultRemotePort = 233
    @JvmStatic const val defaultLocalPort = 1093
    @JvmStatic const val defaultRoute = "bypass-lan"
    //SSR
    @JvmStatic const val defaultTcpProtocol = "origin"
    @JvmStatic const val defaultObfsMethod = "plain"
    //
    //TODO: should not use hard-coded
    @SuppressLint("SdCardPath")
    @JvmStatic const val baseDir = "/data/data/com.proxy.shadowsocksr/"
    //
    @JvmStatic const val STATUS_CONNECTED = 0
    @JvmStatic const val STATUS_FAILED = 1
    @JvmStatic const val STATUS_DISCONNECTED = 2
    //
    @JvmStatic val lineSept = System.getProperty("line.separator")
}